packages/flutter/lib/src/rendering/proxy_box.dart

```diff
@@ -954,6 +954,26 @@ abstract class CustomClipper<T> {
   String toString() => '$runtimeType';
 }
 
+/// A [CustomClipper] that clips to the outer path of a [ShapeBorder].
+class ShapeBorderClipper extends CustomClipper<Path> {
+  /// Creates a [ShapeBorder] clipper.
+  ///
+  /// The [shapeBorder] argument must not be null.
+  const ShapeBorderClipper({
+    @required this.shapeBorder
+  }) : assert(shapeBorder != null);
+
+  final ShapeBorder shapeBorder;
+
+  /// Returns the outer path of [shapeBorder] as the clip.
+  @override
+  Path getClip(Size size) => shapeBorder.getOuterPath(Offset.zero & size);
+
+  @override
+  bool shouldReclip(covariant ShapeBorderClipper oldClipper)
+    => oldClipper.shapeBorder != shapeBorder;
```

Comment:
is there any benefit to using `=>` here rather than `return`? `return` would be more consistent with other code I think

Reply:
done